### DISCUSSION TITLE
Is it possible to send to iOS using the FCM delivery method?

### DISCUSSION THREAD

**Discussion by berserkia** [excid3/noticed#321] [General]
I've followed the steps detailed here https://rnfirebase.io/messaging/usage/ios-setup to configure Firebase to also send to iOS. The app returns a device_token that I'm saving, but delivering via FCM always returns "BadDeviceToken". 

Has anyone else used the FCM method to deliver to both Android and iOS, or do I need to specifically use the iOS delivery method?

**Comment by t2** [comment]
@berserkia in your mobile app, how are you getting the token for the device?

**Comment by deepsystm** [comment]
> I've followed the steps detailed here https://rnfirebase.io/messaging/usage/ios-setup to configure Firebase to also send to iOS. The app returns a device_token that I'm saving, but delivering via FCM always returns "BadDeviceToken".
> 
> Has anyone else used the FCM method to deliver to both Android and iOS, or do I need to specifically use the iOS delivery method?

+1
have the same question
